content/sensu-go/6.1/release-notes.md

```diff
@@ -68,6 +69,50 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
+- ([Commercial feature][172]) Added [`batch_buffer`, `batch_size`, and `batch_workers` attributes][181] to the PostgresConfig type so operators can optimize PostgreSQL latency and throughput.
+- ([Commercial feature][172]) Added TLS configuration to the cluster resource so you can specify additional CA certificates and insecure mode.
+- ([Commercial feature][172]) Added a `types` query parameter for listing [authentication providers][176] and [secrets providers][177] via the API.
+- ([Commercial feature][172]) The Alpine-based Docker image now has multi-architecture support with support for the linux/386, linux/amd64, linux/arm64, linux/arm/v6, linux/arm/v7, linux/ppc64le, and linux/s390x platforms.
+- The backend flag [`--api-request-limit`][173] is now available to configure the maximum API request body size (in bytes).
+- In the [REST API][174], most configuration resources now support the PATCH method for making updates.
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][172]) Improved logging for OIDC authentication providers. Also added [`disable_offline_access` OIDC spec attribute][180], which allows OIDC providers to return a refresh token so users can refresh their access tokens.
+- ([Commercial feature][172]) Added indexed field and label selectors to the PostgreSQL event store to improve performance for PostgreSQL event store queries with field and label selectors.
+- Added Prometheus transformer for extracting metrics from check output using the Prometheus Exposition Text Format.
+- Added the [`output_metric_tags` attribute][182] for checks so you can apply custom tags to enrich metric points produced by check output metric extraction.
+- A warning is now logged when you request a dynamic runtime asset that does not exist.
+- The trusted CA file is now used for agent, backend, and sensuctl asset retrieval.
+- Per-entity subscriptions (such as `entity:entityName`) are always available for agent entities, even you remove subscriptions via the entities API.
+
+**FIXES:**
+
+- ([Commercial feature][172]) Fixed a bug in `sensuctl dump` that allowed polymorphic resources (e.g., secrets providers and authentication providers) to dump other providers of the same type.
+- ([Commercial feature][172]) Check output is no longer truncated in the event log file when the max output size is set and the PostgreSQL event store is enabled.
+- ([Commercial feature][172]) Sensuctl prune now handles multi-file/multi-url input correctly.
+- ([Commercial feature][172]) Fixed a bug where PostgreSQL errors could cause the backend to panic.
+- ([Commercial feature][172]) Fixed a bug where PostgreSQL would refuse to store event with a negative check status.
+- The backend will no longer start if the web UI TLS configuration is not fully specified.
+- The agent entity is now included in data passed to the STDIN for the command process.
+- Proxy entities used in round robin check requests are no longer stale.
```

Comment:
I feel like this needs a reworking -- it sounds weird to me as written.

Looks like it refers to https://github.com/sensu/sensu-go/pull/4008, where I think Nikki's first description better captures the change:

"Ensures that the interval/cron round robin schedulers do not use stale proxy entities when scheduling check requests."

Maybe something like

```suggestion
Improved check scheduling to prevent stale proxy entity data when using cron or round robin schedulers.
```